Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getWriteConcern.txt

MongoDB\\Collection::getWriteConcern()
======================================

.. meta::

>    :description: Retrieve the write concern for a MongoDB collection using the `getWriteConcern()` method.

Reviewer: ```suggestion
   :description: Retrieve the write concern of a collection with the MongoDB PHP Library.
```
